inquire/src/prompts/path_select/action.rs

```diff
@@ -0,0 +1,73 @@
+use crate::{
+    ui::{Key, KeyModifiers},
+    InnerAction, InputAction,
+};
+
+use super::config::PathSelectConfig;
+
+/// Set of actions for a MultiSelectPrompt.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum PathSelectPromptAction {
+    /// Action on the value text input handler.
+    FilterInput(InputAction),
+    /// Moves the cursor to the option above.
+    MoveUp,
+    /// Moves the cursor to the option below.
+    MoveDown,
+    /// Moves the cursor to the page above.
+    PageUp,
+    /// Moves the cursor to the page below.
+    PageDown,
+    /// Moves the cursor to the start of the list.
+    MoveToStart,
+    /// Moves the cursor to the end of the list.
+    MoveToEnd,
+    /// Toggles the selection of the current option.
+    ToggleCurrentOption,
+    /// Selects all options.
+    SelectAll,
+    /// Deselects all options.
+    ClearSelections,
+    /// Navigates deeper in file tree
+    NavigateDeeper,
+    //// Navigated higher in file tree
+    NavigateHigher,
+}
+
+impl InnerAction<PathSelectConfig> for PathSelectPromptAction {
+    fn from_key(key: Key, config: &PathSelectConfig) -> Option<Self> {
+        if config.vim_mode {
```

Comment:
I have to do some proper vim mode, because the current way is kind of bad. e.g. on vim_mode there is no way for users to type `k` or `j` in the input.

Let's not use vim_mode on this new prompt for now, and once I fix it for the others I can add it here.

Reply:
If the config won't change the behavior, it might be better to remove it entirely so it doesn't confuse the users